include/ofi_mr.h

```diff
@@ -54,10 +54,16 @@
 int ofi_open_mr_cache(uint32_t version, void *attr, size_t attr_len,
 		      uint64_t flags, struct fid **fid, void *context);
 
+struct ofi_mr_hmem_ipc_info {
+    uint8_t  handle[IPC_HANDLE_SIZE];
+    void     *mapped_addr;
+};
```

Comment:
Move fields directly into ofi_mr_info and avoid separate structure.  It's only used within ofi_mr_info anyway

Reply:
fixed.